### PR TITLE
Enable frontend unit testing

### PR DESCRIPTION
**Pull request by kenan-rhoton**
Step 1 of #81.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,17 @@ While working on backend stuff (Clojure), please keep a terminal open at *all* t
...
 
+While working on frontend stuff (Javascript), please keep a terminal open at *all* times with the following command running:
+
+`docker-compose run frontend yarn test-refresh`
+
 This will give you instant feedback through the tests.
 
+If you don't want to use docker (and have Leiningen/Yarn installed), you can use the non-docker versions of the commands, which **must** be executed from within the respective directories (`backend`and `frontend` respectively):
+
+`lein test-refresh`
```
Comment (Masclins) [inline README.md]: You really had this already commited? xD

File: fulltest.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@ set -e # Stop the script on any failure or error
 docker-compose -f docker-compose.yml -f docker-compose.e2e.yml down
 docker-compose -f docker-compose.yml -f docker-compose.e2e.yml build
 docker-compose run backend lein test
+docker-compose run frontend yarn test
```
Comment (Masclins) [inline fulltest.sh]: This is for Travis?

File: README.md (hunk 1 above)
Comment (kenan-rhoton) [reply README.md]: Yes I did 🙃

File: fulltest.sh (hunk 2 above)
Comment (kenan-rhoton) [reply fulltest.sh]: No, I'm working on travis, this is for the local "fulltest".

File: .travis.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ jobs:
         - stage: Unit Tests
           script:
               - docker-compose run backend lein test
+              - docker-compose run frontend yarn test
```
Comment (kenan-rhoton) [inline .travis.yml]: THIS is for Travis 🙃

**Pull request merged**